stress/common/src/main/java/alluxio/stress/common/FileSystemParameters.java

```diff
@@ -46,13 +55,28 @@ public FileSystemClientType convert(String value) {
   }
 
   /**
-   * @return FileSystemClientType of this bench
    * Converts from String to FileSystemClientType instance.
+   * @return FileSystemClientType of this bench
    */
   public static class FileSystemParametersReadTypeConverter implements IStringConverter<ReadType> {
     @Override
     public ReadType convert(String value) {
       return ReadType.fromString(value);
     }
   }
+
+  /**
+   * Converts from String to a valid FileSystemWriteType String.
+   * @return FileSystemWriteType of this bench
+   */
+  public static class FileSystemParametersWriteTypeConverter implements IStringConverter<String> {
+    @Override
+    public String convert(String value) {
+      if (value.equals("MUST_CACHE")  || value.equals("CACHE_THROUGH")  || value.equals("THROUGH")
```

Comment:
Can you use the values from `WriteType`?

Also seems our convention is like `FileSystemClientType.fromString()`, your method should be named `fromString()` and follow the same pattern.

Reply:
Since we have a "ALL" parameter here, should I define the value as WriteType? My idea is that because I will delete the "all” in the batch task runner, I will make the change in that PR.